config/application.rb

```diff
@@ -1,6 +1,19 @@
 require File.expand_path('../boot', __FILE__)
 require File.expand_path('../preinitializer', __FILE__)
-require 'rails/all'
+require "rails"
+%w(
+  active_record
+  action_controller
+  action_view
+  action_mailer
+  active_job
+  sprockets
```

Comment:
`rails/all` rescues because gems may not be available; we don't need to do that, so a straight set of `require "active_record/railtie"` etc will probably read better.

Exploding this require is perfectly normal... the generator does it for you if you opt out of one of the frameworks.